cmd/describe/cluster/cmd.go

```diff
@@ -324,11 +324,47 @@ func controlPlaneConfig(cluster *cmv1.Cluster) string {
 func clusterInfraConfig(cluster *cmv1.Cluster, clusterKey string, r *rosa.Runtime) string {
 	var infraConfig string
 	if cluster.Hypershift().Enabled() {
-		infraConfig = fmt.Sprintf(""+
-			"Nodes:\n"+
-			" - Compute (workers):           %d\n",
-			cluster.Nodes().Compute(),
-		)
+		nodePools, err := r.OCMClient.GetNodePools(cluster.ID())
```

Comment:
I hate that we have two API endpoints for essentially the same stuff: nodepools vs machinepools. The original proposal when this project started was to abstract those differences from the API to avoid this type of duplication... not anyone's fault (apart from maybe mine), just ranting :stuck_out_tongue:

Reply:
I know :). There was some back-and-forth on this.
The tradeoff was to keep the "nodepools"  API in OCM as an implementation detail and consolidate the "machinepool" or "hosted machinepools" at ROSA CLI level.

Trying to use a single "machinepool" API from OCM for both scenarios would trigger an interesting refactoring both in classic and hypershift scenarios.

This was a tradeoff to keep progressing until we can progress, but I see what you mean (I hope that at least I can share the "rationale" about the "why").